recipe/meta.yaml

```diff
@@ -1,35 +1,38 @@
-{% set version = "3.2.3" %}
+{% set version = "3.2.4" %}
 
 package:
   name: scs
   version: {{ version }}
 
 source:
-  - url: https://github.com/bodono/scs-python/archive/refs/tags/{{ version }}.tar.gz
-    sha256: 7592805810fb6da09955dc45322b2d376bca993928c972f472f1a5352821ddcd
+  - url: https://github.com/bodono/scs-python/archive/refs/tags/{{ version }}.post1.tar.gz
+    sha256: bb509507d857906eb95d17d5824138e48e7c75d1a692d0aa7256d899c7e703c5
   # include submodule (not in github tarball due to dear-github/dear-github#214)
   - git_url: https://github.com/cvxgrp/scs.git
-    git_rev: ac6840a3b3264950e6c300264cbf3937e0bcc6c5
-    folder: scs
+    git_rev: 9024b8ccc1bba6ee797440fb22354cadb9c81839
+    folder: scs_source
 
 build:
   number: 0
   script:
-    - export BLAS_LAPACK_LIB_PATHS="$PREFIX/lib"    # [unix]
-    - export BLAS_LAPACK_LIBS="lapack:cblas:blas"   # [unix]
-    - {{ PYTHON }} -m pip install . -vv
+    # - export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig  # [ppc64le]
+    # - echo "python = '${PREFIX}/bin/python'" >> ${CONDA_PREFIX}/meson_cross_file.txt  # [ppc64le]
+    - MESON_ARGS_REDUCED="$(echo $MESON_ARGS | sed 's/--buildtype release //g')"  # [build_platform != target_platform]
+    - {{ PYTHON }} -m pip install . -vv -Csetup-args=${MESON_ARGS_REDUCED// / -Csetup-args=}  # [build_platform != target_platform]
+    - {{ PYTHON }} -m pip install . -vv  # [build_platform == target_platform]
```

Comment:
Please remove debug output, align selectors, and perhaps add a comment about why we need to strip that option (if you found that being used somewhere else in conda-forge already, just copy the comment from there too; if you came up with it yourself, the comment shouldn't be hard ;-)).

Reply:
Thanks, I will clean this up. It worked locally for me on Linux. I wasn't expecting the trouble with cross-compiling when I opened the PR. I will mark it as a draft for now.